install.md

```diff
@@ -7,6 +7,34 @@ main_link: True
 
 # Install or upgrade
 
+## mozregression-gui
+
+For the graphical interface, we want to provide a simple installation process:
+
+- **Windows**:
+  Just download the latest **mozregression-gui.exe** file from the
+  [github releases]. This is an installer, so once the file is downloaded
+  double-click on it and follow the installation process.
+
+- **Linux 64**:
+  We provide a frozen application (that includes python itself) in the
+  [github releases]. You can try it out by downloading the
+  **mozregression-gui.tar.gz**, decompress it somewhere and run the
+  mozregression-gui executable file.
+
+Please note that these installers are news - please report any bug you can
```

Comment:
I'd rephrase this and the subsequent paragraph a bit. Maybe emphasize that the whole thing is new and consolidate content. Here's what I'd suggest:

Please note that the mozregression GUI (including the installation process) is at a very early stage of development -- please report any bug you find. A graphical installation process for MacOS X is planned. You can still install the GUI directly [from github](https://github.com/mozilla/mozregression/blob/master/gui/README.rst) if your OS is not supported or if you have issues with the prebuilt packages.